src/main/java/org/alliancegenome/curation_api/interfaces/base/crud/BaseReadIdentifierControllerInterface.java

```diff
@@ -24,7 +24,7 @@ public interface BaseReadIdentifierControllerInterface<E extends AuditedObject>
 
 	@GET
 	@Path("/{identifierString}")
-	@JsonView(View.FieldsOnly.class)
+	@JsonView(View.FieldsAndLists.class)
```

Comment:
Might this cause issues for other endpoints?

Reply:
We tend to use different views and overwrite these endpoints where lists would be a problem.  The alternative is to keep this as is but then we would need to create CDS, Transcript, and Exon views and overwrite all those endpoints to use those views (which seems overkill at this point).